suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -20,24 +20,38 @@ import { notificationsActions } from '@suite-common/toast-notifications';
 import { transactionsActions } from '../transactions/transactionsActions';
 import { selectTransactions } from '../transactions/transactionsReducer';
 import { accountsActions } from './accountsActions';
-import { selectAccountByKey, selectAccounts } from './accountsReducer';
+import {
+    selectAccountByKey,
+    selectAccounts,
+    selectPortfolioTrackerAccounts,
+} from './accountsReducer';
 import { ACCOUNTS_MODULE_PREFIX } from './accountsConstants';
 import { selectBlockchainHeightBySymbol } from '../blockchain/blockchainReducer';
 
 export const disableAccountsThunk = createThunk(
     `${ACCOUNTS_MODULE_PREFIX}/disableAccountsThunk`,
-    (_, { dispatch, extra, getState }) => {
+    (
+        { excludePortfolioTrackerAccounts = false }: { excludePortfolioTrackerAccounts?: boolean },
+        { dispatch, extra, getState },
+    ) => {
         const {
             selectors: { selectEnabledNetworks },
         } = extra;
         const accounts = selectAccounts(getState());
         const enabledNetworks = selectEnabledNetworks(getState());
+
+        const portfolioAccountKeys: AccountKey[] = excludePortfolioTrackerAccounts
+            ? selectPortfolioTrackerAccounts(getState()).map(a => a.key)
+            : [];
+
         // find disabled networks
         const disabledNetworks = networksCompatibility
             .filter(n => !enabledNetworks.includes(n.symbol) || n.isHidden)
             .map(n => n.symbol);
         // find accounts for disabled networks
-        const accountsToRemove = accounts.filter(a => disabledNetworks.includes(a.symbol));
+        const accountsToRemove = accounts.filter(
+            a => disabledNetworks.includes(a.symbol) && !portfolioAccountKeys.includes(a.key),
```

Comment:
It can be simplified by using `imported` flag that is true for portfolio tracker accounts.

Reply:
great idea!

Reply:
makes it pretty simple indeed
4d181fd4de